gui/slick/views/displayShow.mako

```diff
@@ -540,8 +493,13 @@ $(document).ready(function(){
                     ${file_size}
                 % endif
             </td>
+            ## <% date = (sbdatetime.sbdatetime.sbfdate(sbdatetime.sbdatetime.convert_to_setting(network_timezones.parse_date_time(epResult['airdate'], show.airs, show.network))), 'never')[int(epResult['airdate']) == 1] %>
+            <% date = sbdatetime.sbdatetime.convert_to_setting(network_timezones.parse_date_time(epResult['airdate'], show.airs, show.network)) %>
```

Comment:
Yes, I agree about having a common function to format the date.
I'm always finding something else to do on SickRage, so it's hard to have a prioritized list.